101-azurefirewall-create/azuredeploy.parameters.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "virtualNetworkName": {
+      "value": "yourvnetname"
+    },
+    "vnetAddressPrefix": {
```

Comment:
remove the rest of the parameters since you have defaultValues for them

Reply:
Done